### ISSUE TITLE
Giveth Swag ETHBCN

### ISSUE THREAD

**Issue by rodricastillo9013** [Giveth/GIVeconomy#593]
hello team, I create this task, since I think we could work on the Swag for ETHBCN with time and advance this work, what do you think? @laurenluz @markoprljic

**Comment by markoprljic** [comment]
Yes I think this is great idea! I think we need something fresh! Jes could help maybe?

**Comment by laurenluz** [comment]
Yeah! Awesome idea. We can hand out swag tokens at the conference perhaps.

Maybe we should make some new designs to add to the swag shop?

**Comment by laurenluz** [comment]
Hey @RodriCastillo#1415 @markop#2007 and whoever else in design wants to weigh in. For ETH BCN we do have a booth, so it would be worthwhile setting it up with about $5k of swag to hand out.

Would be great if we could spruce up our swag designs for the store... Maybe just adding 2 more t-shirt designs. I would also love to try to print more stickers.

I would love to see a teaser for GIVpower... something using the rocket ship or like a "coming soon". Open to other ideas, but let's try to keep it simple and not make too much work for ourselves :)

**Comment by rodricastillo9013** [comment]
<img width="516" alt="Captura de Pantalla 2022-05-24 a la(s) 17 27 41" src="https://user-images.githubusercontent.com/61791380/170253273-3ccafe18-b43d-4fc7-93cc-e54c5eb19702.png">
<img width="481" alt="Captura de Pantalla 2022-05-24 a la(s) 17 42 49" src="https://user-images.githubusercontent.com/61791380/170253387-99b8c72a-6f3f-4bc7-9cd7-57d337954e99.png">
great, we also thought with lauren that summer hats and bags for people would be nice.

**Comment by laurenluz** [comment]
@markoprljic we should keep an eye on this post: https://forum.giveth.io/t/should-we-sponsor-ethbarcelona/562/20

Hopefully Manu gives us some details about what the expect for the booth, etc.

**Comment by markoprljic** [comment]
Few things to keep in mind when designing new swag:

1. Art on the frontside should be visually appealing and attractive
2. Try to go beyond the existing designs and explore new ideas 
3. Make one special ETHBCN design only for this event (should include text ETH Barcelona 2022)
4. Try to add Giveth logo on the back of the shirt somewhere close to the top (people read what's on your back)
5. If the logo is on the front as well, make sure it is in the upper chest area (or shoulder sleeve) so that it is visible when having video calls

Look up for inspiration on other swag shops and Web3 brands. 

@rodricastillo9013

**Comment by markoprljic** [comment]
We should also create linktr.ee page and generate qr code which leads to that page, and print stickers with this we code.

That qr code should aslo be included in all our presentations and can be used elsewhere.

**Comment by markoprljic** [comment]
Few more notes on existing products.

<img width="555" alt="Screenshot 2022-05-28 at 15 43 52" src="https://user-images.githubusercontent.com/5704285/170828139-ad466d75-b14d-43cc-a748-7db7bfcd900a.png">

- Logo is not visible, should probably be white or yellow
- the text in general is no readable on this dark shirt
- instead of type treatment we could have designed the smiley and legible text below, simpler maybe
- consider using different shirt color (purple, yellow, white, cyan blue or similar)

<img width="555" alt="Screenshot 2022-05-28 at 15 47 16" src="https://user-images.githubusercontent.com/5704285/170828230-72a52f33-2f5a-4f1d-9834-d333b309e8f5.png">

- this print could probably be bigger, the ration of the graphic vs. shirt is not that good
- the design itself could have been more interesting with the rainbow sample 
- overall we should make designs with more impact

Things to consider:

- use different colors of shirts, not just black
- try to make more mature designs, not too serious but professional looking
- don't be constrained with the brand guidelines and existing assets, try making something new!
- think out of the box, try all over print idea
- consider the fact that at conferences we want to stand out in the crowd of hundreds of other teams with amazing swag designs they have

**Comment by markoprljic** [comment]
Also, colourful socks! Let's make that 😊

**Comment by rodricastillo9013** [comment]
yeah!
<img width="471" alt="Captura de Pantalla 2022-05-30 a la(s) 12 23 02" src="https://user-images.githubusercontent.com/61791380/171022791-e46ec874-ae00-466f-8626-c8f98a4e0829.png">
I thought maybe we could make a special design for the team, maybe a beach shirt, with fun colors.

**Comment by laurenluz** [comment]
Hey fam! @Danibelle is going to help coordinate and get us an order of swag ready for ETH BCN 😄 we're getting close to the dates so I think it would be great if we could place an order sometime next week.

Puts a bit of pressure/deadline for the designs, but we can also spend time updating the swag store in the coming weeks.

**Comment by Danibelle** [comment]
> We should also create linktr.ee page and generate qr code which leads to that page, and print stickers with this qr code.
> 
> That qr code should aslo be included in all our presentations and can be used elsewhere.

This is brilliant!!

**Comment by Danibelle** [comment]
Tracking order quantities and costs here!
https://docs.google.com/spreadsheets/d/1BcyAhEKWAAPlNo7X-6KZgV01VnM_LRazhvemDxxbdNI/edit#gid=0

**Comment by laurenluz** [comment]
Ok! I uploaded the new designs to printful and make a big order to the giveth house yesterday. I added the shirt & bags to the store, but didn't add the bucket hat because they were $19 which is significantly more than our other swag items. We can add them later though, if we want.

We got one shirt design:
![image](https://user-images.githubusercontent.com/75490971/172633601-b5c0927e-a1f0-4e1b-9fe0-e8f298c22eae.png)

Shirt quantities & sizes:
![image](https://user-images.githubusercontent.com/75490971/172633428-8212df92-08d3-4fe5-aaf3-f0f4faa07977.png)

For bags, we got two designs, 50 of each:
![image](https://user-images.githubusercontent.com/75490971/172633776-5d0070d6-e796-4c35-869c-15acdeae7378.png)
![image](https://user-images.githubusercontent.com/75490971/172633839-72a49d68-081e-45dc-aa16-c8f9bf387e4a.png)

We also got 30 bucket hats, one design:
![image](https://user-images.githubusercontent.com/75490971/172634029-eaa12a4a-7bb2-4db0-83c5-7fd6640c8547.png)

They should arrive to the Giveth house before July 1, put the order to go to Franco.

The order totaled about $4300 charged to griff. I will sent the details to freshelle so she can help create the reimbursement request.

**Comment by rodricastillo9013** [comment]
amazing @laurenluz  thanks

**Comment by laurenluz** [comment]
Sticker designs are here! https://drive.google.com/drive/folders/1hnTeq43SblPrPa6omB0OmmZurMV_WkgY

**Comment by c0ric0ri** [comment]
@rodricastillo9013 @Danibelle I assume this issue can be closed off, or is there anything outstanding to add? Please update the status accordingly, thanks!
